website/templates/project/settings.mako

```diff
@@ -378,8 +378,31 @@
                     <div class="panel-heading clearfix">
                         <h3 class="panel-title">Email Notifications</h3>
                     </div>
+                    % if node['mailing_list_enabled'] or user['is_admin']:  ## Begin Project Mailing Lists Settings
```

Comment:
I'm not sure that I know what "new project settings section" you're referring to.

<img width="1160" alt="screen shot 2016-05-20 at 23 03 58" src="https://cloud.githubusercontent.com/assets/5659262/15446025/43e30458-1edf-11e6-88f8-e3b5be77498f.png">